src/lib/types/minions.ts

```diff
@@ -356,8 +356,9 @@ export interface UnderwaterAgilityThievingTaskOptions extends ActivityTaskOption
 
 export interface PuroPuroActivityTaskOptions extends MinigameActivityTaskOptions {
 	quantity: number;
-	implingID: number | null;
+	implingName: string | null;
```

Comment:
You really cant store names in options like this. Its not necessary, its a waste of db space, and its unsound (if the name changes). IDs *only* in task options.

Reply:
Resolved this now